easybuild/easyconfigs/c/Cython/Cython-0.26-GCCcore-6.4.0-Python-2.7.13-bare.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'PythonPackage'
+
+name = 'Cython'
+version = '0.26'
+pyver = '2.7.13'
+pyshortver = '2.7'
+pysubver = '-bare'
+versionsuffix = '-Python-%s%s' % (pyver, pysubver)
```

Comment:
Not dancing around it at all.  '-bare' indicates that it doesn't include the Python needed later.   That it, it indicates that this isn't a standalone module but must be paired with a full Python.  If you prefer '-build' to '-bare' we can do that.   But the Python version has to be specified (I can't use the 2.7 Cython with 3.6)..